src/compiler/crystal/syntax/ast.cr

```diff
@@ -1092,21 +1092,21 @@ module Crystal
 
   class OffsetOf < ASTNode
     property offsetof_type : ASTNode
-    property instance_var : ASTNode
+    property offsetof_exp : ASTNode
```

Comment:
Let's use `offset` instead of `offset_exp` everywhere. I thought that might conflict with something else, but I don't think so.

Sorry for the going and coming with this! 🙏

Reply:
No prob, I'm learning a lot about the compiler codebase with this simple patch.